Client/Client.Qt/Style/StyleBasic.cpp

```diff
@@ -221,6 +223,7 @@ void init_StyleBasic(int scale)
 
     initPxValues(scale);
 
+    _defaultLoadingSpinner    = {st::spinnerLineColor, 1.5708, 16, px8, px2, px12, 0.0314159, 0.8};
```

Comment:
What does 1.5708 mean?
Use comments with magical numbers or put it into variable.
<cmath> has M_PI for number PI.

Reply:
1.5708 is revolutions per second